modules/govuk_env_sync/files/govuk_env_sync.sh

```diff
@@ -338,7 +338,9 @@ function restore_elasticsearch {
 
 function dump_postgresql {
   # Check which postgres instance the database needs to restore into
-  if [ "${database}" == 'content_data_api_production' ]; then
+  if [ "${database}" == 'account-api_production' ]; then
+    db_hostname='account-api-postgresql-primary'
+  elif [ "${database}" == 'content_data_api_production' ]; then
```

Comment:
Hmm one of the things that comes to mind here is that when we flip this we'll find the existing db instance isn't sync'd. I wonder if there's a way we can get this to run both in old world and new world as this gradually fans out.

Reply:
Resolved by #11343 ⭐ thanks!